### PR TITLE
Clear data persistence

### PR DESCRIPTION
**Pull request by GreihMurray**
Clearing data persistence on reload of page

### CODE REVIEW

**Review by abeiler7 — APPROVED** [review]

File: app/templates/base.html [hunk 1]
```diff
@@ -51,6 +51,14 @@
                 text-align: left;
                 float: right;
                 overflow-y: auto;
+          }
```
Comment (bmacgavin) [inline app/templates/base.html:54]: Didn't this commit already happen in your last PR?

**Comment by bmacgavin** [comment]
I feel like some of these changes had already happened in a previous PR. Would there ever be an issue with double committing the same thing to GitHub?

File: app/templates/base.html (hunk 1 above)
Comment (GreihMurray) [reply app/templates/base.html:54]: Yeah, I had based this branch off of that one prior to the pull request going through, then rebased onto development

**Comment by GreihMurray** [comment]
There shouldnt be. It didnt raise any flags when I first made the pull request

**Review by bmacgavin — APPROVED** [review]

**Pull request merged**